docs/README.md

```diff
@@ -0,0 +1,13 @@
+# Morphir Documentation
+The purpose of the document is to provide a detailed explanation of how various Morphir code artefacts work.
+It also documents the standard coding practices adopted for the Morphir project.
+Finally, it provides a step by step walk-throughs on how various Morphir components are build.
+
+## Who this Guide Designed For
+Morphir has different components for different users. 
+
+1. **[Morphir User's Guide](users-guide)** - Business users who would like to learn how to use Morphir for modelling business logic.
+2. **[Morphir Developer's Guide](developers-guide)** - New joiners to the Morphir project to get up to speed the various building blocks of Morphir.
+3. **[Morphir Contributor's Guide](contribution-guide)** - Existing team members intending to improve their abilities on Language Design concepts.
+
```

Comment:
I find the descriptions here confusing. Developer vs contributor is both working on Morphir but one is a new joiner? Which guide is for developers who want to integrate Morphir into their applications?

I'm not sure what's the best way to organize this but we definitely need guides for:
- How to model business concepts using Morphir.
- How to integrate Morphir tools into the SDLC.
- How to extend existing or build new Morphir tools.